src/main/scala/iog/psg/cardano/CardanoApiImpl.scala

```diff
@@ -63,12 +64,12 @@ private class CardanoApiImpl(baseUriWithPort: String)(implicit ec: ExecutionCont
    * @inheritdoc
    */
   override def createRestoreWallet(
-                                    name: String,
-                                    passphrase: String,
-                                    mnemonicSentence: MnemonicSentence,
-                                    mnemonicSecondFactor: Option[MnemonicSentence] = None,
-                                    addressPoolGap: Option[Int] = None
-                                  ): Future[CardanoApiRequest[Wallet]] = {
+    name: String,
```

Comment:
Why do these get aligned differently to the base API version? The base API version was good.

Reply:
hmm weird, updatePassphrase seems identical, and was formatted properly, maybe its because of name param in a new line and has some space there, will check this